hvplot/tests/testplotting.py

```diff
@@ -58,3 +68,10 @@ def setUp(self):
             raise SkipTest('entrypoints for plotting.backends was added '
                            'in pandas 0.25.1')
         pd.options.plotting.backend = 'hvplot'
+
+
+@pytest.mark.skipif(skip_polar, reason="polars not installed")
+def test_plot_supports_polars():
+    dfp = pl.DataFrame(pd._testing.makeDataFrame())
```

Comment:
```suggestion
def test_plot_supports_polars():
    pl = pytest.importorskip("polars")
    dfp = pl.DataFrame(pd._testing.makeDataFrame())
```

Alternative way (also needs import removal also)

Reply:
Made the change, even if I'd say that having the marker above the test definition helps distinguish tests when looking at many of them, while an inline skip is easy to miss.

Reply:
Yep, I wouldn't have had a problem if you kept what you had before.